### ISSUE TITLE
Holochain fails to start if proxy server is down

### ISSUE THREAD

**Issue by alastairong** [holochain/holochain#508]
Issue
- Holochain was failing to start. Error message was something related to KitsuneP2P
- We noticed that proxy server had been stopped
- Once proxy server was started, Holochain could start again

Also found when proxy server is started but busy:

here's a sample error:
```
Dec 23 15:46:58 hpos holochain[27636]: FATAL PANIC PanicInfo {
Dec 23 15:46:58 hpos holochain[27636]:     payload: Any,
Dec 23 15:46:58 hpos holochain[27636]:     message: Some(
Dec 23 15:46:58 hpos holochain[27636]:         Could not initialize Conductor from configuration: HolochainP2pError(OtherKitsuneP2pError(TransportError(Other(ConnectionClosed(ConnectionClose { error_code: SERVER_BUSY, frame_type: None, reason: b"" }))))),
Dec 23 15:46:58 hpos holochain[27636]:     ),
Dec 23 15:46:58 hpos holochain[27636]:     location: Location {
Dec 23 15:46:58 hpos holochain[27636]:         file: "crates/holochain/src/bin/holochain/main.rs",
Dec 23 15:46:58 hpos holochain[27636]:         line: 133,
Dec 23 15:46:58 hpos holochain[27636]:         col: 10,
Dec 23 15:46:58 hpos holochain[27636]:     },
Dec 23 15:46:58 hpos holochain[27636]: }
```

Environment:
- HoloPort following channel 651

**Comment by freesig** [comment]
I have also seen this issue. We might need to make the network a little more resilient when the proxy cannot be reached. @neonphog

**Comment by freesig** [comment]
I've created a backlog item for this.

**Comment by artbrock** [comment]
This issue prioritized in our current sprint now.

**Comment by abe-njama** [comment]
@neonphog might the fix to this issue have been in this PR: #602 ?

**Comment by ThetaSinner** [comment]
@abe-njama @alastairong any update about whether this is resolved by `holochain` 0.1?
